crates/srs/src/voice_stream.rs

```diff
@@ -33,15 +33,18 @@ impl VoiceStream {
     pub async fn new(
         client: Client,
         addr: SocketAddr,
-        recv_voice: bool,
-    ) -> Result<Self, io::Error> {
+        game_source: Option<mpsc::UnboundedReceiver<GameMessage>>
+    ) -> Result<Self, io::Error>
+    {
+        let recv_voice = game_source.is_some();
+
         let tcp = TcpStream::connect(addr).await?;
         let (sink, stream) = Framed::new(tcp, MessagesCodec::new()).split();
 
         let a = Box::pin(recv_updates(stream));
-        let b = Box::pin(send_updates(client.clone(), sink, recv_voice));
+        let b = Box::pin(send_updates(client.clone(), sink, game_source));
 
-        let udp = UdpSocket::bind("127.0.0.1:0").await?;
+        let udp = UdpSocket::bind("0.0.0.0:0").await?;
```

Comment:
I am wondering whether it is worth the effort of checking if the SRS server address is a local address and only if not using `0.0.0.0` instead of `127.0.0.1` to not "expose" the socket unnecessarily. But might be not worth the effort - what do you think?

Reply:
If there is a use for it, i'd say let's implement it, but I don't really see a use for it. Would it be to avoid accidentally send things to the wrong server?